### ISSUE TITLE
"Visualize Vector As Raster" missing behaviour.

### ISSUE THREAD

**Issue by gab3d** [opentoonz/opentoonz#4153]
When using the "Visualize Vector As Raster" option the selected (vector) level doesn't visualize as vector, as the [manual states here](https://opentoonz.readthedocs.io/en/latest/drawing_animation_levels.html#adjusting-the-viewer-visualization).

![imagen](https://user-images.githubusercontent.com/18104091/142016518-675c0030-3d3a-4bb9-ba19-e4c745f905d6.png)

I think that missing behaviour of the function potentially makes it way less powerful, as a user wanting to edit a vector level will have to switch the option OFF to be able to see it with good quality. But in doing so, the Viewer can potentially get slow (supposing the reason to activate it in first place was to speed it up, because of a slow performance, most probably due to many complex vector levels displayed).

Perhaps the code is still there, and could be reactivated?
As always, thanks for listening!

**Comment by DarrenTAnims** [comment]
You can still edit the level using the vector tools (control point editor, magnet tool, etc), so you don't have to switch off the option to edit the current level.

But you are right that the current level isn't "visualised" as a vector level usually is. It would be nice if it was and it would be clearer which parts of the drawing were in the current level.

**Comment by gab3d** [comment]
@DarrenTAnims 
yes, as you say, you can still edit strokes in vectorial form, but without seeing them properly, so for rough changes it'd be fine, but when you need to make more detailed tweaks, certainly it'd not be enough, and that's the case where the original behaviour of the feature comes in handy. 👍

**Comment by andeon** [comment]
@RodneyBaker This appears to be fixed by PR https://github.com/opentoonz/opentoonz/pull/5793. Feel free to close the issue.
